library/src/main/res/layout/exo_playback_control_view.xml

```diff
@@ -58,7 +58,7 @@
     <TextView android:id="@id/exo_position"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:textSize="14sp"
+      android:textSize="13sp"
```

Comment:
Non-Stock android ROMS,
and apps with overridden fonts

For instance
```
FontsOverride.setDefaultFont(this, "DEFAULT", "fonts/proximanova_regular.ttf");
```
overrides the default font.